heroku.yml

```diff
@@ -0,0 +1,13 @@
+setup:
+  addons:
+    - plan: heroku-postgresql
+      as: DATABASE
```

Comment:
I think this is the default

Reply:
yes, do you think it's best to remove it?

Reply:
Usually, I don't set default values, but I'm ok either way